pom.xml

```diff
@@ -268,7 +268,7 @@
                             <goal>generate</goal>
                         </goals>
                         <configuration>
-                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/core/api/payment_transactions_gateway_api/internal/v1/_openapi.json.tpl</inputSpec>
+                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-notifications-service/v1/_openapi.json.tpl</inputSpec>
```

Comment:
You replaced the payment-transaction-gateway spec by mistake 😃

Reply:
Thank you giovanni